[DM-15392] Initial kube deployment for dax-webserv.

cbanek opened the pull request:
Initial work at dax-webserv deploys.

Contains:
1) A service, deployment, and ingress rules for nginx-ingress.
2) Persistent volumes and their claims for the dax webservice.
3) Helper scripts to get started.

cbanek committed:
[DM-15392] Initial kube deployment for dax-webserv.

Initial work at dax-webserv deploys.

Contains:
1) A service, deployment, and ingress rules for nginx-ingress.
2) Persistent volumes and their claims for the dax webservice.
3) Helper scripts to get started.

brianv0 reviewed (approved):
Looks good. Same question as I had in albuquery.

cbanek committed:
[DM-15392] Fix port problems, and URL ingress.

1) Webserv runs on port 5000, not 8080.  So update that all around.
2) Update the URL to be /api/image/v1 for imageserv.
3) Add a new endpoint for metaserv at /api/meta/v1.

cbanek committed:
[DM-15392] Add integration test script.

This script tests imageserve and metaserve to ensure that things
are behaving well in the PDAC.
